commands/commands.go

```diff
@@ -95,4 +95,5 @@ func init() {
 	NanoboxCmd.AddCommand(DestroyCmd)
 	NanoboxCmd.AddCommand(StartCmd)
 	NanoboxCmd.AddCommand(StopCmd)
+	NanoboxCmd.AddCommand(versionCmd)
```

Comment:
Created it as its own command to be less intrusive to the main NanoboxCmd (rather than a flag)